package.json

```diff
@@ -17,7 +17,8 @@
     "prebuilds/**",
     "bindings/node/*",
     "queries/*",
-    "src/**"
+    "src/**",
+    "*.wasm"
```

Comment:
Is anything besides the wasm file actually valuable?

Reply:
For the VSCode extension (https://github.com/daivinhtran/biome-vscode/pull/1), we don't need anything other than the wasm file. But I can't really speak for other use cases.

For example, the node binding is built and installed on `node_modules/tree-sitter-gritql/prebuilds/darwin-arm64/tree-sitter-gritql.node` during an npm install. If we remove `prebuilts/**` from this list, it excludes the node binding.

These are other examples we can use as guidance:
https://github.com/tree-sitter/tree-sitter-python/blob/26855eabccb19c6abf499fbc5b8dc7cc9ab8bc64/package.json#L26-L33
https://github.com/tree-sitter/tree-sitter-javascript/blob/58404d8cf191d69f2674a8fd507bd5776f46cb11/package.json#L26-L33